### ISSUE TITLE
Texture size causes blurry images when imported

### ISSUE THREAD

**Issue by ultrawidegamer** [Yellow-Dog-Man/Resonite-Issues#2332]
### Is your feature request related to a problem? Please describe.

Since adding the relative and max texture size settings, if utilized then all images when imported will be blurry even if they are just meant to be temporarily viewed before deleted (memes, textures, images of text, etc)  

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/147885362/017fc44f-2d01-4597-b7a1-775741443aae)

### Describe the solution you'd like

I would like to have an override option in the settings to override both the max and relative texture size settings until the images have been saved.

### Describe alternatives you've considered

So far i have been just manually opening all imported images and checking the direct load option on the image if it is meant to be just temporarily viewed

### Additional Context

related to the below added feature
https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1658

### Requesters

ultrawidegamer

**Comment by Frooxius** [comment]
This is a bit of a weird scenario - what exactly constitutes an image that's been "saved"? And would that really be enough here?

E.g. say you save it and you spawn it out again because you want to show it to someone - it'll be blurry then, leading to the same issue. It feels like the this is not really appropriate solution, because it's a bit too "hacky" - it would solve the problem in a very specific case, but leave it in other situations.

I don't know what would be the best way to automate this though - you can technically set a minimum size on the imported texture if you need stuff like text to be legible and so on. You can also set the minimum texture size in settings for cases like this.

**Comment by ultrawidegamer** [comment]
I dont think there is a good solution to all of the scenarios, but i also dont think there has to be... this would be more of a creature comfort feature so that the need to override the size of a texture would become WAY less frequent. as for example it would stop me from having to hit the direct load option dozens of times per day versus just once in a while if i want to save an image.

**Comment by Frooxius** [comment]
Problem is if there are issues with the approach, we'll likely get more issues and reports because of that that we'll have to deal with - we're generally going to be very hesitant to invest our time into implementing such features.

We'd need to figure out a good plan of action here and think about the problem some more.

**Comment by stawze** [comment]
Images imported into a world should definitely be treated differently than textures on materials

**Comment by shadowpanther** [comment]
> Images imported into a world should definitely be treated differently than textures on materials

The problem is, imported images _are_ textures on materials. That's how the rendering works.

**Comment by Banane9** [comment]
> > Images imported into a world should definitely be treated differently than textures on materials
> 
> The problem is, imported images _are_ textures on materials. That's how the rendering works.

For fresh imports, it could be a "simple" check between local and resdb, I suppose - however as froox pointed out, that would leave the issue of images spawned from the inventory.

**Comment by stawze** [comment]
> > Images imported into a world should definitely be treated differently than textures on materials
> 
> The problem is, imported images _are_ textures on materials. That's how the rendering works.

Yeah that makes sense. Maybe pictures need an import option that adds some metadata? Although people could abuse that to bypass the texture limits. Maybe that metadata could then be removed when it is applied to a material
